component/componenttest/otelchecker.go

```diff
@@ -15,6 +15,13 @@ import (
 	"go.opentelemetry.io/collector/component"
 )
 
+func checkScraperLogs(reader *sdkmetric.ManualReader, receiver component.ID, scraper component.ID, scrapedLogRecords, erroredLogRecords int64) error {
```

Comment:
ditto.

Reply:
updated in https://github.com/open-telemetry/opentelemetry-collector/pull/12036/commits/fc21ef2ff66531442987223c30621d6febbcb2b2